app/src/parser/parser.ts

```diff
@@ -19,6 +20,8 @@ const lru = require('tiny-lru');
 const datastore = new Datastore();
 const hgtCache = lru(300);
 
+const downloadQueue = new Queue(10, 10);
```

Comment:
I think it would be great to have a constructor like
`new Queue({max_tasks: 10, delay: 10});`
 What do you think ?

Reply:
Yes, now that I think of, but it is an already published package